.github/workflows/continuous-integration.yml

```diff
@@ -159,9 +163,13 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
-      - run: ./docker-compose-run-as-entrypoint.sh print npm ci
+      - run: mkdir print/node_modules print/.nuxt
+
+      - run: sudo chown -R 1000:1000 .cache print/node_modules print/.nuxt
+
+      - run: CI=false ./docker-compose-run-as-entrypoint.sh print npm ci
 
-      - run: ./docker-compose-run-as-entrypoint.sh print npm run build
+      - run: CI=false ./docker-compose-run-as-entrypoint.sh print npm run build
```

Comment:
Ich musste gerade nachschauen was `CI=false` schon wieder macht. Vielleicht wäre es verständlicher, einfach dann `./docker-compose-run-as-entrypoint.sh --user 1001 print npm run build` zu schreiben, wenn man es im CI auch wirklich braucht? Von 15 Aufrufen (die ja in Zukunft wieder weniger werden könnten wenn wir mehr auf renovate Groups setzen) sind jetzt in 3 `CI=false` drin.

Reply:
Ich verwende jetzt statt dem magischen flag einfach die env variable $USER_ID, die es geben sollte.
Es kommt noch  --rm --no-deps --entrypoint dazu, und irgendwann vergisst man einfach, etwas zu copy pasten.
(Passiert mir auch immer wieder, und dann muss ich wieder diese docker container aufräumen, die wieder herumliegen)

Reply:
Find ich gut so, merci vielmal